src/segger/data/io.py

```diff
@@ -963,21 +1009,24 @@ def build_pyg_data_from_tile(
         transcripts_df["token"] = token_encoding  # Store the integer tokens in the 'token' column
         data["tx"].token = torch.as_tensor(token_encoding).int()
         # Handle additional embeddings lazily as well
-        if not self.embedding_df.empty:
+        if self.embedding_df is not None and not self.embedding_df.empty:
```

Comment:
Same bug as above: when the user was not specifying the embedding, an exception was raised.